lib/Wikia/tests/Localisation/LCStoreDBIntegrationTest.php

```diff
@@ -1,48 +0,0 @@
-<?php
-namespace Wikia\Localisation;
-
-/**
- * @group Integration
- */
-class LCStoreDBIntegrationTest extends \WikiaDatabaseTest {
```

Comment:
Why do we remove the test class?

Reply:
SQLite unfortunately has trouble with `INSERT ... ON DUPLICATE KEY UPDATE SET foo = VALUES(foo)`syntax, so it's not really possible to write an isolated integration test now—if we could use mysql in docker or similar solution, it'd be fine.